templates/keymanager/resources/vkcs_keymanager_secret.md.tmpl

```diff
@@ -11,18 +11,11 @@ description: |-
 {{ .Description }}
 
 ## Example Usage
-### Simple secret
+### Secret from locally generated private key
```

Comment:
Private key for TERMINATED_HTTPS loadbalancer listener

Reply:
done